db/schema.rb

```diff
@@ -10,908 +10,908 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_02_09_033203) do
```

Comment:
Lets not alter the db:schema file generated by rails.  I think only the version number change should happen with the migration that is added. so lets revert the other changes.

Reply:
Reverted this. PTAL